hilti/toolchain/src/compiler/cfg.cc

```diff
@@ -174,6 +174,9 @@ GraphNode CFG::_addBlock(GraphNode predecessor, const Nodes& stmts, const Node*
         else if ( auto* try_catch = c->tryAs<statement::Try>() )
             predecessor = _addTryCatch(predecessor, *try_catch);
 
+        else if ( auto* switch_ = c->tryAs<statement::Switch>() )
+            predecessor = _addSwitch(predecessor, *switch_);
+
```

Comment:
Nit: these aren't really ordered, at least not alphabetically. Let's either do that, or move this up to after e.g., `If` where we have more branching stuff.